packages/terra-status-view/package.json

```diff
@@ -34,8 +34,8 @@
   "peerDependencies": {
     "react": "^16.2.0",
     "react-dom": "^16.2.0",
-    "terra-base": "^3.7.0",
-    "terra-mixins": "^1.15.0"
+    "terra-base": "^3.11.0",
+    "terra-mixins": "^1.16.0"
```

Comment:
lets remove terra-mixins as a peer dependency since it's already listed as a dependency and remove terra-base as a dependency and leave it a peer dependency.

Reply:
Here ya go! 
https://github.com/cerner/terra-core/pull/1583/commits/1d1ed4ab4d19d863458730fa575296dbb8888d16